Modules/_testcapimodule.c

```diff
@@ -4051,6 +4051,86 @@ raise_SIGINT_then_send_None(PyObject *self, PyObject *args)
 }
 
 
+static PyObject *
+test_pyobject_fastcall(PyObject *self, PyObject *args)
+{
+    PyObject *func, *argstuple;
+    PyObject **stack;
+    Py_ssize_t nargs;
+
+    if (!PyArg_ParseTuple(args, "OO!", &func, &PyTuple_Type, &argstuple)) {
+        return NULL;
+    }
+
+    nargs = PyTuple_GET_SIZE(argstuple);
+    if (nargs) {
+        stack = &PyTuple_GET_ITEM(argstuple, 0);
+    }
+    else {
+        stack = NULL;
+    }
+    return _PyObject_FastCall(func, stack, nargs);
```

Comment:
Unfortunately this can't be used for testing the case `nargs == 0`, `stack != NULL`. Maybe set `stack` to `NULL` only when pass `None` as args?